webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyFileChecksum.java

```diff
@@ -0,0 +1,62 @@
+package com.box.l10n.mojito.entity;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Index;
+import javax.persistence.Table;
+
+/** Entity that stores the checksum of a translated file downloaded via a third party sync. */
+@Entity
+@Table(
+    name = "third_party_sync_file_checksum",
+    indexes = {
+      @Index(
+          name = "I__THIRD_PARTY__CHECKSUM__REPOSITORY_ID__FILE_NAME__LOCALE_ID",
+          columnList = "repository_id, locale_id, file_name",
+          unique = true),
+    })
+public class ThirdPartyFileChecksum implements Serializable {
+
+  @EmbeddedId private ThirdPartyFileChecksumId thirdPartyFileChecksumId;
```

Comment:
A surrogate key (auto-incrementable int) is preferable for maintainability/updates - makes it much easier to link to this data from other tables and is generally more flexible for future changes.